### PR TITLE
Upgrade scenario

### PR DESCRIPTION
**Pull request by JumaX**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Added new molecule scenario confluent-kafka-kerberos-customcerts-rhel to test usage of confluent-kafka instead of confluent-server.  Will be utilized in future upgrade testing scenarios.

Have updated the following scenarios to have 3 zookeeper instances instead of 1, to facilitate upgrade testing in the future:

rbac-scram-custom-rhel
rbac-scram-provided-debian

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

All scenarios have been run and validated via molecule verify.

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/verify.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      vars:
+        file_path: /etc/kafka/server.properties
+        property: sasl.enabled.mechanisms
+        expected_value: GSSAPI
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_package.yml
+      vars: 
+        package_name: confluent-kafka-2.12-5.5.0-1.noarch
```
Comment (domenicbove) [inline roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/verify.yml:18]: this package name will need to be updated on each branch, is there a way to not include the "5.5.0-1"

File: roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+      - kerberos_server
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: zookeeper1
```
Comment (domenicbove) [inline roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml:17]: can we test colocated zk and kafka w this one as well?

Comment (JumaX) [reply roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml:17]: @domenicbove I have another bug open that I plan on working on after this, regarding upgrades and uninstall packages.  This directly relates to having co-located components, and I will implement the checks in that work.  So on this PR, I think it's best to leave it out.

Comment (JumaX) [reply roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml:17]: There is a bigger issue here, which is that when we upgrade we leave older packages around which can cause conflicts.    This directly impacts colocated components, as if you remove packages while another service is running, you could have problems.  So it makes sense to do this as a separate issue.  I would suggest you either wait to merge your PR until that work is complete or go ahead and merge now, as it's a fairly minor change.

Comment (JumaX) [reply roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml:17]: It is for setting up the upgrade testing in 600-post, however the colocated components issue is new, and I had actually planned to fix the issue your PR resolves in the bug I filed for this.  So if you want, you can go ahead and merge this, as its pretty easy to check as is, or you can wait until the additional work is complete around the bug fix.

**Review by domenicbove — APPROVED** [review]
LGTM

**Pull request merged**